Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM python:3.11-slim-buster
+FROM python:slim
```

Comment:
Why? I like to pin the python version, so it won't automatically start using v12 without me testing it. I'm not too concerned about slim vs. slim-buster, but what is your reasoning for slim over slim-buster? It appeared slim-buster supported more architectures when I was looking before.

Reply:
slim uses debian bullseye and buster uses debian buster

Reply:
Why do you think slim should be used over slim-buster? And please update to pin python 3.11.